static/temp/foods.json

```diff
@@ -0,0 +1,259 @@
+        { "name": "Fiber", "percentage": "40%", "weight": "120g" }
+      ],
+      "nutrients": ["Protein", "carbohydrate", "water", "vitamin", "Fats", "Fiber", "Minerals"],
+      "allFilters": {
+        "Life Stage": [
+          {
+            "name": "Infant",
+            "icon": "OilBarrelOutlinedIcon"
+          },
+          {
+            "name": "Children",
+            "icon": "DirectionsCarFilledOutlinedIcon"
+          },
+          {
+            "name": "Male",
+            "icon": "OilBarrelOutlinedIcon"
+          },
+          {
+            "name": "Female",
+            "icon": "DirectionsCarFilledOutlinedIcon"
+          },
+          {
+            "name": "Pregnant",
+            "icon": "OilBarrelOutlinedIcon"
+          },
+          {
+            "name": "Location",
+            "icon": "DirectionsCarFilledOutlinedIcon"
+          }
+        ],
+        "age": ["9-13 Years", "14-18 Years", "19-30 Years", "31-50 Years", "51-70 Years"]
+      },
+      "nutrientTableHeaders": {
+        "allNutrients": [
+          "Protein",
+          "carbohydrate",
+          "water",
+          "vitamin",
+          "Fats",
+          "Fiber",
+          "Minerals"
+        ],
+        "value": "% Daily Value",
+        "rdi": "RDI"
+      },
+      "nutrientTableContent": [
+        {
+          "contentName": "Total carbohydrate  4g",
+          "data": [
+            { "mineral": "Carbohydrate  2g", "value": "17.67%", "rdi": "130g" },
+            { "mineral": "Sugar  2g", "value": "17.67%", "rdi": "15.8g" },
+            { "mineral": "Starch  2g", "value": "17.67%", "rdi": "12.6g" }
+          ]
+        },
+        {
+          "contentName": "Protein  11g",
+          "data": [{ "mineral": "Protein 2g", "value": "1.61%", "rdi": "46g" }]
+        }
+      ]
+    },
+    "recipes": [
+      {
+        "name": "Banana Walnut Bread",
+        "description": "Flour, baking soda, ripe banana, vanilla extract",
+        "duration": {
+          "prepTime": {
+            "hours": 0,
+            "minutes": 10
+          },
+          "cookTime": {
+            "hours": 0,
+            "minutes": 20
+          },
+          "totalTime": {
+            "hours": 0,
+            "minutes": 30
+          }
+        },
+        "instructions": [
+          {
+            "stepNumber": 1,
+            "description": "Wash and cut your banana inside a neat bowl and mix thoroughly till it’s smoothly combined",
+            "videoUrl": ""
+          },
+          {
+            "stepNumber": 2,
+            "description": "Dice the spinach into smaller pieces and pour inside your mixed banana and stir thoroughly.",
+            "waitTime": {
+              "hours": 0,
+              "minutes": 10
+            }
+          }
+        ],
+        "reviews": ["5", "5", "5", "4"],
+        "ingredients": [
+          {
+            "name": "Banana",
+            "quantity": 1,
+            "unit": "10g"
```

Comment:
Why `unit` is `10g`?

Reply:
Match the values as per the [Figma design](https://www.figma.com/file/gmopGK9vLw5MATERWlWT9W/Forkfacts-with-material-theme?node-id=1360%3A65840&t=hMLYLBlO9Z4zT3y9-4).

Make this change in `groceryLists` as well.